### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by Srinivas-raju** [community/community#58637] [Programming Help]
### Select Topic Area

Question

### Body

I have two branches: master, develop
I created file in develop branch and want to ignore it while merging so i have created .gitattributes  file in root directory both branches.

end executing the below code

git config --global merge.ours.driver true
git checkout master
echo "path/file_to_ignore merge=ours" >> .gitattributes
git merge develop

but it's not working can you pls suggest me

**Comment by airtower-luna** [comment]
What exactly is your goal here? You can't exempt certain paths/commits from a merge. And wanting to merge two branches but ignore one of them sounds odd at least. :thinking: 

If you only want to take certain commits, look at [`git cherry-pick`](https://git-scm.com/docs/git-cherry-pick). If you actually merge after cherry-picking you'll end up with duplicate commits, though.
